code_samples/sequelize/tags/sequelize.js

```diff
@@ -38,7 +38,7 @@ client.once('ready', () => {
 	Tags.sync();
 });
 
-client.on('message', async (msg) => {
+client.on('message', async(msg) => {
```

Comment:
Space here and no parenthesis around `msg`. I think it'd also be a good idea to change all instances of `msg` to `message`, since that's what we use everywhere else in the guide.

Reply:
Once again, eslint file dictates - but changing the variable names, sure.